### ISSUE TITLE
Ice and Fire 1.18 village structure generation issues along with Compatibility Datapack refusing to enable.

### ISSUE THREAD

**Issue by qqbone** [jtorleonstudios/HUB-Better-Villages#23]
Ice and fire seems to heavily inject its village structure aggressively when used with the Bettervillager mod. It tends to force other modded structures as well like shrines from paraglider. I have tested it without Ice and Fire and it'll generate fine. I've tried this on multiple versions of Better Villages as well from 1.0.2 to 3.0.0.
![vanillawithbettervillages](https://user-images.githubusercontent.com/92418035/230745256-a5373a87-d4b4-4908-9d76-baabc38b6dd4.png)
![villagesvanillaiceandfire](https://user-images.githubusercontent.com/92418035/230745257-16a8fc8f-fc58-486f-b289-68ef88c4b2c0.png)
![iceandfireplainsvillage](https://user-images.githubusercontent.com/92418035/230745265-b55d5716-0f25-41f3-b2ea-d60961d7a739.png)
![iceandfireicevillage](https://user-images.githubusercontent.com/92418035/230745267-31615370-e14a-46f7-837a-2b04f5df982b.png)

I read in another submission that you can manage structure generation configs. Issue is on the either on  Ice and  Fire putting the weight at 0 doesn't disable the entirety of a chance that they will spawn but solves a fair amount of it overloading with Scribe buildings. 

There's also an issue with it breaking Paraglider's Goddess Statues on generation too, but that's probably because of Ice and Fire not playing nice with Bettervillages.

Last issue is that while playing on version 1.0.2 of BetterVillages I cannot seem to enable your compatibility datapack at all. Is there a cut off for a certain version?

**Comment by jtorleonstudios** [comment]
Hi, the images in your post are not the result of the compatibility datapack. None of these structures should be present. 

I did not test with Better Villages 1.0.2, only with the latest version, so 3.0.0. but normally there should be no problem, where did you place the datapack?

**Comment by qqbone** [comment]
I'm aware it's not the compatibility datapack. I sadly found it after I made the post looking for solutions for two very famous mods but I'm not running into the issue of not being able to enable the datapack to solve most the issues.

I cannot for the life of me enable it on my server. It's in a heavier modpack so I'm unaware why it's refusing to be enabled. Maybe it's just the version of 3.0 Ferritlibrary or lower version.
I've tried in the general datapack, the datapack that is in the world save, tried manually extracting it to global packs (removed this process) and used Paxi. Should just be a simple drag, drop, enable.
